engine/apps/user_management/subscription_strategy/free_public_beta_subscription_strategy.py

```diff
@@ -61,12 +61,10 @@ def _calculate_phone_notifications_left(self, user):
         day_start = now.replace(hour=0, minute=0, second=0, microsecond=0)
         calls_today = PhoneCallRecord.objects.filter(
             created_at__gte=day_start,
-            represents_alert_group__channel__organization=self.organization,
```

Comment:
👍

Reply:
we remove it because user can be a member of single organization and organization filter is excess, isn't it?

Reply:
right